mininext/node.py

```diff
@@ -75,11 +77,13 @@ def startShell(self):
             opts += 'u'
         # bash -m: enable job control
         # -s: pass $* to shell, and make process easy to find in ps
-        cmd = ['mxexec', opts, 'bash', '-ms', 'mininet:' + self.name]
-        self.shell = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=STDOUT,
+        cmd = [ 'mxexec', opts, 'env', 'PS1=' + chr( 127 ),
+                'bash', '--norc', '-is', 'mininet:' + self.name ]
+        master, slave = pty.openpty()
+        self.shell = Popen(cmd, stdin=slave, stdout=slave, stderr=slave,
                            close_fds=True)
```

Comment:
Looks like `close_fds=False` now in mininet -- should it be updated here as well?